.scrutinizer.yml

```diff
@@ -4,3 +4,4 @@ imports:
 tools:
     external_code_coverage:
         runs: 5 # No Code Coverage on PHP 5.2 and HHVM
+        timeout: 2700 # 45 minutes
```

Comment:
Oh right. Makes sense then. My test suites are normally much smaller so take less time. 45 mins is probably good then. :)